syncopy/datatype/methods/arithmetic.py

```diff
@@ -165,10 +171,64 @@ def _parse_input(obj1, obj2, operator):
         lgl = "Syncopy object, scalar or array-like"
         raise SPYTypeError(operand, varname="operand", expected=lgl)
 
-    return operand_dat, opres_type, operand_idxs
+    return baseObj, operand, operand_dat, opres_type, operand_idxs
+
+def _perform_computation(baseObj,
+                         operand,
+                         operand_dat,
+                         operand_idxs,
+                         opres_type,
+                         operator):
+    """
+    Coming soon...
+    """
+
+    # Prepare logging info in dictionary: we know that `baseObj` is definitely
+    # a Syncopy data object, operand may or may not be; account for this
+    if "BaseData" in str(operand.__class__.__mro__):
+        opSel = operand._selection
+    else:
+        opSel = None
+    log_dct = {"operator": operator,
+               "base": baseObj.__class__.__name__,
+               "base selection": baseObj._selection,
+               "operand": operand.__class__.__name__,
+               "operand selection": opSel}
+
+    # Create output object
+    out = baseObj.__class__(dimord=baseObj.dimord)
+
+    # Wrap operator in lambda function
+    if operator == "+":
+        operation = lambda x, y : x + y
+
+    if __acme__:
+        try:
+            dd.get_client()
+            parallel = True
+        except ValueError:
+            parallel = False
+
+    # Perform actual computation
+    opMethod = SpyArithmetic(operand_dat, operand_idxs, operation=operation,
+                             opres_type=opres_type)
+    opMethod.initialize(baseObj,
+                        out._stackingDim,
+                        chan_per_worker=None,
```

Comment:
Arithmetic should be almost trivially parallelisable over channels no? However probably not really needed..

Reply:
Yupp, should be. The only thing that's sabotaging it is the possible existence of in-place selections in one (or both) objects. Then the channel parallelization setup becomes complicated. Definitely not impossible, but unless users request it, I thought we could table that one for later

Reply:
Agreed :+1: